etc/kayobe/environments/ci-aio/automated-setup.sh

```diff
@@ -6,7 +6,7 @@ cat << EOF | sudo tee -a /etc/hosts
 10.205.3.187 pulp-server pulp-server.internal.sms-cloud
 EOF
 
-if [ sudo vgdisplay | grep -q lvm2 ]; then
+if [ "sudo vgdisplay | grep -q lvm2" ]; then
```

Comment:
If you just want to test the exit code you can do this:
```suggestion
if sudo vgdisplay | grep -q lvm2; then
```

Reply:
I think your change tests the string rather than executing the command.

Reply:
You're right it was passing the conditional on the string, thanks for the fix!